src/components/Payment/LegalNote/LegalNote.tsx

```diff
@@ -46,7 +46,7 @@ const LegalNote = () => {
           if (isInTrial) {
             return (
               <Trans
-                i18nKey={`legal-notes.trial.period-${offerPeriod as SubscriptionPeriodType}`}
+                i18nKey={`offer-standard-consent-copy.trial.period-${offerPeriod as SubscriptionPeriodType}`}
               >
```

Comment:
the whole LegalNote component should be removed as it is not longer used. Also all imports (in Payments components)

Reply:
Will remove also LegalCopy